client/me/help/help-unverified-warning/index.jsx

```diff
@@ -52,18 +52,18 @@ class HelpUnverifiedWarning extends Component {
 				resendState: RESEND_IN_PROGRESS,
 			} );
 
-			user.sendVerificationEmail()
+			user.sendVerificationEmail( this.props.userId, this.props.nonce )
 				.then( () => {
 					const nextResendState = RESEND_SUCCESS;
 
 					this.setState( { resendState: nextResendState } );
 					notices.success( resendStateToMessage( nextResendState ) );
 				} )
-				.catch( () => {
+				.catch( ( error ) => {
 					const nextResendState = RESEND_ERROR;
 
 					this.setState( { resendState: nextResendState } );
-					notices.error( resendStateToMessage( nextResendState ) );
```

Comment:
I suspect we can now remove this [case statement](https://github.com/Automattic/wp-calypso/blob/48e1e5c872c95a588efa06444760e764d498b75c/client/me/help/help-unverified-warning/index.jsx#L43-L44) since we no longer need that error message.